macros/Compat.ejs

```diff
@@ -698,7 +698,7 @@ if (!compatData) {
 traverseFeatures(compatData, depth, '');
 
 if (features.length > 0) {
-    output = '<div class="bc-data">';
+    output = `<div class="bc-data" data-bcd-id="{identifier}">`;
```

Comment:
If you make this `id` instead of `data-bcd-id`, I think it should work. Kuma bleaches out attributes that aren't explicitly
allowed per https://github.com/mozilla/kuma/blob/master/kuma/wiki/constants.py. This will give you an id containing "." but apparently that is legal https://stackoverflow.com/questions/70579/what-are-valid-values-for-the-id-attribute-in-html

Also, I think query would be better, since you want the full bcd query like "html.element.video" and not just "video".

```suggestion
    output = `<div class="bc-data" id="${query}">`;
```

Reply:
Done. Thanks! 
Now I guess I need to figure out how to run tests.